python/cuml/test/test_nearest_neighbors.py

```diff
@@ -408,22 +416,42 @@ def test_nearest_neighbors_sparse(nrows, ncols,
 
     a = cp.sparse.random(nrows, ncols, format='csr', density=density,
                          random_state=32)
+    b = cp.sparse.random(nrows, ncols, format='csr', density=density,
+                         random_state=33)
 
-    logger.set_level(logger.level_info)
-    nn = cuKNN(metric=metric, n_neighbors=n_neighbors, algorithm="brute",
+    if metric == 'jaccard':
+        a = a.astype('bool').astype('float32')
+        b = b.astype('bool').astype('float32')
+
+    nn = cuKNN(metric=metric, p=2.0, n_neighbors=n_neighbors,
+               algorithm="brute", output_type="numpy",
                verbose=logger.level_debug,
                algo_params={"batch_size_index": batch_size_index,
                             "batch_size_query": batch_size_query})
     nn.fit(a)
 
-    cuD, cuI = nn.kneighbors(a)
+    import time
+
+    start = time.time()
+    cuD, cuI = nn.kneighbors(b)
+    print("cuML took %s" % (time.time() - start))
```

Comment:
This looks like debug code that was left over. Good idea not to pollute the test log.
```suggestion
```